tests/models/awards/BaseContract-test.js

```diff
@@ -77,70 +78,71 @@ describe('BaseContract', () => {
         });
     });
     describe('awardType', () => {
-       it('should return the idv type for the idv category', () => {
-           const mockIdv = Object.assign({}, mockApi, {
-               category: null
-           });
-           const idv = Object.create(BaseContract);
-           idv.populate(mockIdv);
+        it('should return the idv type for the idv category', () => {
+            const mockIdv = Object.assign({}, mockApi, {
+                category: null
+            });
+            const idv = Object.create(BaseContract);
+            idv.populate(mockIdv);
 
-           expect(idv.awardType).toEqual('mock idv type');
-       });
-       it('should return the contract type otherwise', () => {
-           expect(contract.awardType).toEqual('mock contract type');
-       });
+            expect(idv.awardType).toEqual('mock idv type');
+        });
+        it('should return the contract type otherwise', () => {
+            expect(contract.awardType).toEqual('mock contract type');
+        });
     });
     describe('agencies', () => {
-       it('should only create an awarding/funding agency if it is available in the API response', () => {
-           expect(contract.awardingAgency).toBeTruthy();
-           expect(contract.fundingAgency).toBeFalsy();
-       });
-       it('should format toptier and subtier names', () => {
-          expect(contract.awardingAgency.name).toEqual('Department of Sandwiches');
-          expect(contract.awardingAgency.subtierName).toEqual('Department of Subs');
-          expect(contract.awardingAgency.officeName).toEqual('Office of Cheesesteak');
-       });
+        it('should only create an awarding/funding agency if it is available in the API response', () => {
+            expect(contract.awardingAgency).toBeTruthy();
+            expect(contract.fundingAgency).toBeFalsy();
+        });
+        it('should format toptier and subtier names', () => {
+            expect(contract.awardingAgency.name).toEqual('Department of Sandwiches');
+            expect(contract.awardingAgency.subtierName).toEqual('Department of Subs');
+            expect(contract.awardingAgency.officeName).toEqual('Office of Cheesesteak');
+        });
     });
     describe('Place of Performance', () => {
-        it('should format the regional address', () => {
-            expect(contract.placeOfPerformance.regionalAddress).toEqual('Pawnee, IN 12345');
-        });
-        it('should format the full address', () => {
-           expect(contract.placeOfPerformance.fullAddress).toEqual('Pawnee, IN 12345\nCongressional District: IN-04')
+        it('should be an object with CoreLocation in its prototype chain', () => {
+            const locationObject = Object.create(CoreLocation);
+
+            expect(Object.getPrototypeOf(locationObject)).toEqual(Object.getPrototypeOf(contract.placeOfPerformance));
```

Comment:
A clearer way of testing this would be
```
expect(Object.getPrototypeOf(contract.placeOfPerformance)).toEqual(CoreLocation);
```

Reply:
Can we add prototype tests to the other subobjects (agency/recipient/etc) as well?